### ISSUE TITLE
Can we transfer asset in between spot and futures wallet?

### ISSUE THREAD

**Issue by avatar-lavventura** [sammchardy/python-binance#626]
`client.transfer_spot_to_margin()` function exists, but I coud not find any alternative for the `futures`.

Is there any function for futures as well such as:

`client.transfer_spot_to_future()` or `client.transfer_future_to_spot()`?

**Comment by kimchirichie** [comment]
i dont think so, but I would gladly accept a PR to merge. Would you be able to open up a PR?

i would imagine the code to do something like
requestclient.py:
```
    def post_account_transfer(self, asset: 'str', amount: 'float', transferType: 'int') -> any:
        response = call_sync(self.request_impl.post_account_transfer(asset, amount, transferType))
        self.refresh_limits(response[1])
        return response[0]
```

restapirequestimpl.py:
```
    def post_account_transfer(self, asset, amount, transferType):
        check_should_not_none(asset, "asset")
        check_should_not_none(amount, "amount")
        check_should_not_none(transferType, "transferType")
        builder = UrlParamsBuilder()
        builder.put_url("asset", asset)
        builder.put_url("amount", amount)
        builder.put_url("type", transferType)
        request = self.__create_request_by_post_with_signature("/sapi/v1/futures/transfer", builder)
        def parse(json_wrapper):
            result = Transfer.json_parse(json_wrapper)
            return result
        request.json_parser = parse
        return request
```

**Comment by avatar-lavventura** [comment]
Sure, yes I can open up a PR. I wasn't able to find `restapirequestimpl.py` and `requestclient.py` files in the repo should I make changes in the `client.py` file?

**Comment by avatar-lavventura** [comment]
@kimchirichie Will it be considered to be added as a feature that would be added into the code?  Or by purpose transferring asset(USDT) in between spot and futures is not implemented.

**Comment by kimchirichie** [comment]
@avatar-lavventura sorry i deleted that earlier comment realizing i was referring to a different library. however if there is a PR to add the asset transfer feature, I would gladly review and merge to the library

**Comment by avatar-lavventura** [comment]
@kimchirichie I don't understand why this is not supported on your end, which is a important feature in Futures trading.

**Comment by kimchirichie** [comment]
@avatar-lavventura i only recently became a collaborator of this library. i do not actively develop in this library, only merge low risk PR’s. If you would implement and open a PR for this feature, i will gladly merge this in.

**Comment by kimchirichie** [comment]
i just remembered #505 can you verify this works? i will merge when verified

**Comment by avatar-lavventura** [comment]
@kimchirichie I have tried `client.futures_account_transfer(asset="USDT", amount="1")` but keep getting following error message: `Mandatory parameter 'type' was not sent, was empty/null, or malformed.`. What `type` argument should we provide? do you have any idea?

**Comment by avatar-lavventura** [comment]
@kimchirichie I verify that it works, please the the PR(https://github.com/sammchardy/python-binance/pull/675)
